### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_Eclipse Select Subscriptions_Screen Reader: Narrator is not reading the presence of the checkbox when navigating through the list items

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3349]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator
MAS Violated: MAS1.3.1

**Note: Similar issue is found in the following Screens**
1.	Create Authentication Files

**Repro Steps:** 
1) Start Narrator and Open Java EE - Eclipse application
2) Install 'Azure Toolkit for Eclipse' from Marketplace
3) Tab till tools-->Azure-->Azure Sign in and press enter
4) Select 'Device login' and tab till sign in and press enter
5) Tab till 'Copy and Open' button and press enter
6) Paste the code in the browser and open the Application
7) Verify while navigating through the list items Narrator is reading the presence of the checkboxes

**Actual Result:** 
Narrator is not reading the presence of the checkbox when navigating through the list items

**Expected Result:** 
Narrator should read the control type of the list items as checkboxes

**User Impact:** 
Screen Reader users will not know that they can check or uncheck the list items if screen reader does not read the presence of the list items

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

[Bug_22_Narrator is not reading about the presence of checkbox.pptx](https://github.com/microsoft/azure-tools-for-java/files/3451331/Bug_22_Narrator.is.not.reading.about.the.presence.of.checkbox.pptx)

**Comment by Flanker32** [comment]
@Tanushagottipati This is an external issue of Narrator, and NVDA could read the selected item with the type checkbox, here is a gif.

![3349](https://user-images.githubusercontent.com/12445236/64518453-c80ba300-d324-11e9-9285-2ee177e63fe1.gif)

**Comment by Flanker32** [comment]
* Importance of experience: Little, it is a nice to tell user there is a checkbox when user tab to it in table.
* Quality of experience: not impacted. Since users could use NVDA to get the correct prompt message.
* Alternate solutions: Use NVDA instead of Narrator 
* Evidence: All evidence is provided in this issue.
* [AAT Link](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076)

**Comment by Flanker32** [comment]
@Tanushagottipati This is an external issue of Narrator and we have already linked it to AAT, please help us verify it, thanks!

**Comment by Tanushagottipati** [comment]
External Bug: https://microsoft.visualstudio.com/OS/_workitems/edit/23420155
According to C&AI External Bug process verified with other free screen reader NVDA and the issue not present. As External narrator bug is logged closing this bug
![Screenshot (432)](https://user-images.githubusercontent.com/41109135/64523372-c9839e00-d319-11e9-8146-e05f2dacc10f.png)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749751
